packages/mui-joy/src/Typography/Typography.test.tsx

```diff
@@ -1,8 +1,8 @@
-import * as React from 'react';
+import { ThemeProvider } from '@mui/joy/styles';
+import Typography, { typographyClasses as classes, TypographyProps } from '@mui/joy/Typography';
 import { expect } from 'chai';
+import * as React from 'react';
 import { createRenderer, describeConformance, describeJoyColorInversion } from 'test/utils';
-import Typography, { typographyClasses as classes } from '@mui/joy/Typography';
-import { ThemeProvider } from '@mui/joy/styles';
```

Comment:
Let's not change the order of imports

Reply:
Sorry, forgot to disable the import sorter..